### DISCUSSION TITLE
How to install the last version without composer

### DISCUSSION THREAD

**Discussion by wptechnology** [mpdf/mpdf#1592] [Q&A] [answered]
Hello,
I need to install& use mpdf on a shared webhosting that doesn't have composer nor allowing to install it via SSH.
How can I add mdpf to a PHP project, without composer ?
Thanks.

**Comment by Klap-in** [comment]
Maybe you can use composer at an other machine to download everything, and next transfer all the files to your hosting as normal.

**Comment by finwe** [comment, marked as answer]
It's literally a page of the manual https://mpdf.github.io/installation-setup/using-without-composer.html

**Comment by wptechnology** [comment]
Thanks a lot 👍🏼

 

De : Matěj Humpál ***@***.***>
Répondre à : mpdf/mpdf ***@***.***>
Date : samedi 1 janvier 2022 à 21:27
À : mpdf/mpdf ***@***.***>
Cc : WPTechnology ***@***.***>, Author ***@***.***>
Objet : Re: [mpdf/mpdf] How to install the last version without composer (Discussion #1592)

 

It's literally a page of the manual https://mpdf.github.io/installation-setup/using-without-composer.html

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
